contxt/cli/commands/service_instances.py

```diff
@@ -0,0 +1,56 @@
+import click
+
+from contxt.cli.clients import Clients
+from contxt.cli.utils import print_item, print_table
+
+
+@click.group()
+def service_instances() -> None:
+    """Service Instances."""
+
+
+@service_instances.command()
+@click.pass_obj
+def get(clients: Clients) -> None:
```

Comment:
should we add the `getByServiceId` flow as well since the create requires a service?